src/actionlib/action_client.py

```diff
@@ -612,6 +630,23 @@ def wait_for_server(self, timeout=rospy.Duration(0.0)):
 
         return started
 
+    ## @brief Set timestamp delta for lost goal processing
+    ##
+    ## When processing goals, compare timestamp of the goal and the
+    ## current status message. If the two differ by more than the delta,
+    ## consider the goal LOST.
+    def set_goal_ack_timeout(self, goal_ack_timeout):
+        if goal_ack_timeout:
```

Comment:
Yes, this is a check for `None`
A `goal_ack_timeout` of `0` is valid. That is intended to turn off the check.